### PR TITLE
Add deepmerge to deps ( its used by webpack.check.js ), and touch lint

### PR DESCRIPTION
**Pull request by ponelat**

### CODE REVIEW

**Comment by shockey** [comment]
Hello `.agignore` 😄

File: package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
   "dependencies": {
     "boron": "^0.2.3",
     "classnames": "^2.1.3",
+    "deepmerge": "^1.3.2",
```
Comment (shockey) [inline package.json]: If this is used by webpack, why `dependencies` and not `devDependencies`?

Comment (ponelat) [inline package.json]: good catch

**Comment by shockey** [comment]
@ponelat please make the `devDependency` change and resolve the conflict - besides that, looks good to merge.

thanks!

**Comment by ponelat** [comment]
used in src/index.js https://github.com/swagger-api/swagger-editor/blob/master/src/index.js#L1

**Comment by shockey** [comment]
at long last 👨‍🚀

**Pull request merged**